### ISSUE TITLE
NVDA not reading Bootstrap's tooltip

### ISSUE THREAD

**Issue by lmerlopyx** [nvaccess/nvda#17493]
### Steps to reproduce:
- Go to https://getbootstrap.com/docs/5.3/components/tooltips/#custom-tooltips on Chrome
- Tab into the example button "Custom Tooltip"

### Actual behavior:
only "Custom tooltip, button" is read

### Expected behavior:
Tooltip content exposed via aria-describedby should also be read

#### NVDA installed/portable/running from source:
NVDA Installed

#### NVDA version:
2024.4.1

#### Windows version:
Windows 10 Version 22H2

#### Name and version of other software in use when reproducing the issue:
Chrome Versión 131.0.6778.109 (Build oficial) (64 bits)

#### Does the issue still occur after restarting your computer?
Yes

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
This works as expected in focus mode. It is not intended to be read in browse mode.

**Comment by michaelDCurran** [comment]
More specifically: 
* In focus mode, tabbing to it, the tool tip reads correctly.
* In browse mode, when moving to it with quick nav (f or b) the tooltip reads correctly.
* Arrowing to it in browse mode, the tooltip does not read, but this is deliberate behaviour for aria-describedby.
